### PR TITLE
AO, DS

### PR DESCRIPTION
**Pull request by mrogowski**

### CODE REVIEW

File: src/binding/petsc4py/src/petsc4py/PETSc/AO.pyx
Comment (stefanozampini) [inline src/binding/petsc4py/src/petsc4py/PETSc/AO.pyx:78]: python links go first, then C

File: src/binding/petsc4py/src/petsc4py/PETSc/DS.pyx
Comment (stefanozampini) [inline src/binding/petsc4py/src/petsc4py/PETSc/DS.pyx:24]: not compliant with numpydoc

Comment (mrogowski) [reply src/binding/petsc4py/src/petsc4py/PETSc/DS.pyx]: this is a draft PR, but you can review DS and AO for now

**Review by nocollier — COMMENTED** [review]
Minor changes, looking good.

**Comment by mrogowski** [comment]
Thanks for the review @nocollier, we're ready to merge that once anyone approves

@dalcinl

**Review by nocollier — APPROVED** [review]

**Pull request merged**
